Okta/OktaAuth/OktaApi.swift

```diff
@@ -66,10 +97,13 @@ class OktaApi: NSObject {
 
         return request
     }
+}
+
+class OktaApiImpl: OktaApi {
```

Comment:
OktaApi class name is a bit confusing. It sounds like an entry point API for the SDK. How about to rename:
```suggestion
class OktaRestApi: OktaApi {
```